kernels/armsve/1m/bli_dpackm_armsve256_int_8xk.c

```diff
@@ -35,17 +35,14 @@
 
 #include "blis.h"
 
-#ifdef __ARM_FEATURE_SVE
+#if __has_include(<arm_sve.h>)
```

Comment:
If I understand correctly, the availability of intrinsics really only affects A64fx, right? Can we branch on an A64fx-specific macro here instead of using __has_include which might not be as portable?

Reply:
That's right. `BLIS_FAMILY_A64FX` should work for this case.